cognite/neat/app/api/routers/rules.py

```diff
@@ -28,17 +34,17 @@ def get_rules(
 ) -> dict[str, Any]:
     if NEAT_APP.cdf_store is None or NEAT_APP.workflow_manager is None:
         return {"error": "NeatApp is not initialized"}
-    workflow = NEAT_APP.workflow_manager.get_workflow(workflow_name)
-    if workflow is None:
-        return {"error": f"Workflow {workflow_name} is not found"}
-    workflow_definition = workflow.get_workflow_definition()
-
-    if not file_name:
-        for step in workflow_definition.steps:
-            if step.method == "ImportExcelToRules":
-                file_name = step.configs["file_name"]
-                version = step.configs["version"]
-                break
+    if workflow_name != "undefined":
+        workflow = NEAT_APP.workflow_manager.get_workflow(workflow_name)
+        if workflow is None:
+            return {"error": f"Workflow {workflow_name} is not found"}
+        workflow_definition = workflow.get_workflow_definition()
+        if not file_name:
+            for step in workflow_definition.steps:
+                if step.method == "ImportExcelToRules":
```

Comment:
I would import the `ImportExcelToRules`, just to make sure that the dependency is traceable in code. 
```suggestion
                if step.method == ImportExcelToRules.__name__:
```